js/element-focus/demos/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Установка фокуса на элементы — Element.focus() — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <label>Куда переместить фокус?
+    <select id="focus-selector">
+      <option></option>
+      <option value="name">Имя</option>
+      <option value="email">Email</option>
+      <option value="age">Возраст</option>
+      <option value="specialization">Специализация</option>
+      <option value="nasa">Опыт в NASA</option>
+      <option value="photo">Фото</option>
+      <option value="submit">Кнопка отправки</option>
+    </select>
+  </label>
+  <h1>Регистрация в экспедицию на Марс</h1>
+  <form action="/apply" method="POST">
+    <label>
+      Your name:
+      <input
+        type="text"
+        name="name"
+        id="name"
+      >
+    </label>
+
+    <label>
+      Email:
+      <input
+        type="email"
+        name="email"
+        id="email"
+      >
+    </label>
```

Comment:
```suggestion
    <label>
      Your name:
      <input type="text" name="name" id="name">
    </label>

    <label>
      Email:
      <input type="email" name="email" id="email">
    </label>
```

Вот это было больно )

Reply:
Странно, я брал код из статьи о формах 🤷‍♂️